main.tf

```diff
@@ -60,9 +61,21 @@ resource "aws_codedeploy_app" "default" {
   compute_platform = var.compute_platform
 }
 
+resource "random_id" "deployment_config_suffix" {
+  keepers = {
+    traffic_routing_config       = jsonencode(var.traffic_routing_config)
+    minimum_healthy_hosts        = jsonencode(var.minimum_healthy_hosts)
+    count                        = jsonencode(local.count)
+    deployment_config_name_orgin = jsonencode(module.this.id)
+    compute_platform             = jsonencode(var.compute_platform)
+  }
+
+  byte_length = 2
+}
+
 resource "aws_codedeploy_deployment_config" "default" {
   count                  = local.count
-  deployment_config_name = module.this.id
+  deployment_config_name = "${module.this.id}${module.this.delimiter}${random_id.deployment_config_suffix.hex}"
```

Comment:
I believe we need a separate label for this where the `${random_id.deployment_config_suffix.hex}` can be added as an attribute.

Reply:
I prepare separate label for it